package.json

```diff
@@ -26,10 +41,12 @@
     "build": "gatsby build",
     "develop": "gatsby develop",
     "start": "npm run develop",
+    "deploy": "export NODE_ENV=production && gatsby build && gh-pages -d public -b master",
```

Comment:
This is problematic! It should just be:
`"deploy": "gatsby build && gh-pages -d public",`

 + `gatsby build` is enough for production,
 + Don't deploy to `master`, let `gh-pages` deploy to `gh-pages` branch and change *Settings* on the repo to use `gh-pages` branch for the site.

Reply:
Makes sense changing this